bin/publish

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+# Add the NPM token to the .npmrc file
+echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" >> "$HOME/.npmrc"
```

Comment:
Do we already have this token as an env var within codeship?

Reply:
Yep. It should always be there, and if it's not I'd definitely like to see the script fail.